### PR TITLE
US 1583733 Add missing language IDs - 19

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "output" for exception messages, error messages, and call stack examples.
- Used "console" for PerfView and xperf command examples.
- Used "text" for registry file example.

### CODE REVIEW

File: docs/framework/network-programming/interpreting-network-tracing.md [hunk 1]
```diff
@@ -13,10 +13,9 @@ ms.assetid: ad22b4b8-00af-4778-9cca-cb609ce1f8ff
 # Interpreting Network Tracing
 When network tracing is enabled, you can use tracing to capture calls your application makes to various <xref:System.Net> class members. The output from these calls may be similar to the following examples.  
   
-```  
-[588]   (4357)   Entering Socket#33574638::Send()  
-[588]   (4387)   Exiting Socket#33574638::Send()-> 61#61  
-```  
+`[588]   (4357)   Entering Socket#33574638::Send()`
```
Comment (rpetrusha) [inline docs/framework/network-programming/interpreting-network-tracing.md]: I think it's best to leave this fenced, as it was, and label it as `output`.

File: docs/framework/reflection-and-codedom/dynamically-loading-and-using-types.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Reflection provides infrastructure used by language compilers to implement impli
   
  In the following example, the Visual Basic compiler uses reflection implicitly to call a method on an object whose type is not known at compile time. A **HelloWorld** class has a **PrintHello** method that prints out "Hello World" concatenated with some text that is passed to the **PrintHello** method. The **PrintHello** method called in this example is actually a <xref:System.Type.InvokeMember%2A?displayProperty=nameWithType>; the Visual Basic code allows the **PrintHello** method to be invoked as if the type of the object (helloObj) were known at compile time (early binding) rather than at run time (late binding).  
   
-```  
+```vb
 Imports System
```
Comment (rpetrusha) [inline docs/framework/reflection-and-codedom/dynamically-loading-and-using-types.md]:
```suggestion
```
The line can be deleted, since it's unnecessary.

**Review by rpetrusha — APPROVED** [review]
Thanks, @v-thpra. I left two suggestions for you to consider.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Sorry, Tom. I meant to request changes rather than approve.

**Comment by v-thpra** [comment]
Made the changes. Thanks Ron.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the changes, Tom. I'll merge your PR now.

**Pull request merged**